package.json

```diff
@@ -134,7 +134,7 @@
   "overrides": {
     "loader-utils@<2.0.3": "2.0.4",
     "dset": "3.1.2",
-    "axios": "^1.6.1"
+    "axios": "^1.6.8"
```

Comment:
This was probably done to handle some vulnerabilities in the past because the patch may not be available. Maybe we can try removing this from here. if it doesn't work, we can add it back.

Reply:
I tried removing it Ashok, but the build was failing so I just added back